src/app/footer/footer.component.ts

```diff
@@ -12,6 +13,8 @@ import { GA4GHService } from '../../../src/app/shared/swagger/api/gA4GH.service'
 export class FooterComponent implements OnInit {
   version: string;
   tag: string;
+  public prod = true;
```

Comment:
`prod=true` is probably useless this way. 
the port for the U1_WEBSITE can still come from `https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/shared/dockstore.model.ts`

Reply:
- I originally had the UI1_PORT in the dockstore.model.ts, but that involved changing the dockstore.model.ts (merge conflicts with you)
- I also originally had prod set via the environment files but was told not to.  So change prod to false/true to toggle whether or not the link should be present.